oaknational/Oak-Web-Application
# feat: brushstroke borders

## Description

- Adds `<BrushBorders />` component
- Adds svgs for left, right, top, bottom border lines
- As an enhancement we should inline the svgs in a spritesheet a la #385 

## How to test

1. Go to https://oak-web-application-storybook-189zi8608-oak-national-academy.vercel.app/
2. Click on foundations/BrushBorders
3. You should see the borders on a box as below :point_down: 

## Screenshots

How it should now look:

![image](https://user-images.githubusercontent.com/12934669/181067638-f02e14a9-80b0-40d0-80fd-f6009edeb162.png)


## Checklist
